### PR TITLE
AWS X-Ray integration documentation and cleanup

### PR DESCRIPTION
**Pull request by Hesperide**
- Removes duplicated tagging documentation from X-Ray page
- Highlights X-Ray as an integration in the "Explore" section of our installation docs
- Removes CTA to contact support and cleans up some language

### CODE REVIEW

File: content/en/serverless/installation/_index.md [hunk 1]
```diff
@@ -16,7 +16,9 @@ Install the [AWS integration][1]. This allows Datadog to ingest Lambda metrics f
 
 ### 2. Install the Datadog Forwarder
 
-Install the [Datadog Forwarder Lambda function][2], which is required for ingestion of AWS Lambda traces, enhanced metrics, custom metrics, and logs. **Note**: Skip this step if you already have the Forwarder function installed as part of the [AWS integration][1] CloudFormation stack.
+Install the [Datadog Forwarder Lambda function][2], which is required for ingestion of AWS Lambda traces, enhanced metrics, custom metrics, and logs.
```
Comment (pinkerton) [inline content/en/serverless/installation/_index.md:19]:
```suggestion
Install the [Datadog Forwarder Lambda function][2], which is required to ingest AWS Lambda traces, enhanced metrics, custom metrics, and logs. 
```

File: content/en/serverless/installation/_index.md [hunk 2]
```diff
@@ -16,7 +16,9 @@ Install the [AWS integration][1]. This allows Datadog to ingest Lambda metrics f
 
 ### 2. Install the Datadog Forwarder
 
-Install the [Datadog Forwarder Lambda function][2], which is required for ingestion of AWS Lambda traces, enhanced metrics, custom metrics, and logs. **Note**: Skip this step if you already have the Forwarder function installed as part of the [AWS integration][1] CloudFormation stack.
+Install the [Datadog Forwarder Lambda function][2], which is required for ingestion of AWS Lambda traces, enhanced metrics, custom metrics, and logs. 
+
+**Note**: Skip this step if you already have the Forwarder function installed as part of the [AWS integration][1] CloudFormation stack.
```
Comment (pinkerton) [inline content/en/serverless/installation/_index.md:21]:
```suggestion
**Note**: Skip this step if you already have the Forwarder function installed as part of the [AWS integration][1] CloudFormation Stack.
```

File: content/en/serverless/installation/nodejs.md [hunk 3]
```diff
@@ -278,6 +280,15 @@ exports.handler = async (event) => {
   return response;
 };
 ```
+Learn more about submitting custom metrics [here][3].
```
Comment (pinkerton) [inline content/en/serverless/installation/nodejs.md:283]: Let's make our CTAs more natural and integrated into the copy, like on this APM page: https://docs.datadoghq.com/tracing/app_analytics/?tab=java

File: content/en/serverless/installation/nodejs.md [hunk 4]
```diff
@@ -278,6 +280,15 @@ exports.handler = async (event) => {
...
+
+### Enable the AWS X-Ray Integration
+
+Datadog’s integration with AWS X-Ray allows you to visualize serverless trace data, so you can zero in on the source of any errors or slowdowns, and see how the performance of your functions impacts your users’ experience. Depending on your language and configuration, choose between setting up Datadog APM or the AWS X-Ray integration for your tracing needs. Learn more about configuring distributed tracing [here][4].
```
Comment (pinkerton) [inline content/en/serverless/installation/nodejs.md:287]: Do the APM docs talk about "trace data"? The bigger picture here is visualizing entire end-to-end requests to serverless functions

File: content/en/serverless/installation/python.md [hunk 5]
```diff
@@ -310,6 +312,16 @@ def get_message():
     return "Hello from serverless!"
 ```
 
+Learn more about submitting custom metrics [here][4].
+
+### Enable the AWS X-Ray Integration
+
+Datadog’s integration with AWS X-Ray allows you to visualize serverless trace data, so you can zero in on the source of any errors or slowdowns, and see how the performance of your functions impacts your users’ experience. Depending on your language and configuration, choose between setting up Datadog APM or the AWS X-Ray integration for your tracing needs. Learn more about configuring distributed tracing [here][5].
```
Comment (pinkerton) [inline content/en/serverless/installation/python.md:319]: same feedback about `trace data`

File: content/en/serverless/installation/ruby.md [hunk 6]
```diff
@@ -92,7 +94,13 @@ def some_operation()
     end
 end
 ```
+Learn more about submitting custom metrics [here][9].
+
+### Enable the AWS X-Ray Integration
+
+Datadog’s integration with AWS X-Ray allows you to visualize serverless trace data, so you can zero in on the source of any errors or slowdowns, and see how the performance of your functions impacts your users’ experience. Depending on your language and configuration, choose between setting up Datadog APM or the AWS X-Ray integration for your tracing needs. Learn more about configuring distributed tracing [here][10].
```
Comment (pinkerton) [inline content/en/serverless/installation/ruby.md:101]: same feedback about `trace data`

File: content/en/tracing/serverless_functions/_index.md [hunk 7]
```diff
@@ -20,47 +20,20 @@ Depending on your language and configuration, choose between setting up Datadog
 | Set up using your developer tools without making any changes to your code with Serverless Framework and AWS SAM Integrations. | Install the AWS X-Ray client library for your Lambda runtime. |
 | Support for Python, Node.js, Ruby. |  Support for all Lambda runtimes. |
 
-
-## Organizing your Serverless infrastructure with tags
```
Comment (pinkerton) [inline content/en/tracing/serverless_functions/_index.md:24]: Are these docs present on another page?

File: content/en/tracing/serverless_functions/_index.md [hunk 8]
```diff
@@ -20,47 +20,20 @@ Depending on your language and configuration, choose between setting up Datadog
...
 - [Tracing in a serverless-first environment](#tracing-in-a-serverless-first-environment)
 - [Tracing across AWS Lambda and hosts](#tracing-across-aws-lambda-and-hosts)
 
 #### Tracing in a serverless-first environment
 
-AWS X-Ray is both a backend AWS service and a set of client libraries. The service gives you an Invocation span for your AWS Lambda functions and traces across Amazon API Gateways and message queues.
+AWS X-Ray provides both a backend AWS service and a set of client libraries. Enabling the backend AWS service alone gives you an Invocation span for your AWS Lambda functions as well as traces across Amazon API Gateways and message queues.
 
-The client libraries trace the integrations in your code. If you are using the Datadog APM client library instead of the AWS X-Ray client library to trace and visualize traces, follow the below two steps:
+Both the AWS X-Ray and Datadog APM client libraries trace the integrations in your code. If you are using the Datadog APM client library instead of the AWS X-Ray client library to trace and visualize traces, follow the below two steps:
```
Comment (pinkerton) [inline content/en/tracing/serverless_functions/_index.md:36]: Where do we talk about why you may want to use dd-trace instead?

File: content/en/serverless/installation/nodejs.md [hunk 9]
```diff
@@ -250,6 +250,8 @@ You need to subscribe the Datadog Forwarder Lambda function to each of your func
 
 After you have configured your function following the steps above, you can view metrics, logs and traces on the [Serverless Homepage][2].
 
+### Monitor Custom Serverless Metrics
```
Comment (pinkerton) [inline content/en/serverless/installation/nodejs.md:253]: Do you think this is the right header for this section? Maybe just `Monitor Custom Metrics` or `Monitor Custom Business Metrics` ?

**Review by pinkerton — CHANGES_REQUESTED** [review]
Good edits - left a few comments on small things

File: content/en/tracing/serverless_functions/_index.md (hunk 7 above)
Comment (Hesperide) [reply content/en/tracing/serverless_functions/_index.md:24]: yes

File: content/en/serverless/installation/nodejs.md (hunk 9 above)
Comment (Hesperide) [inline content/en/serverless/installation/nodejs.md:253]:
```suggestion
### Monitor Custom Business Metrics
```

File: content/en/serverless/installation/nodejs.md (hunk 3 above)
Comment (Hesperide) [inline content/en/serverless/installation/nodejs.md:283]:
```suggestion
[Enable custom metric submission][3] to get started.
```

File: content/en/serverless/installation/nodejs.md (hunk 4 above)
Comment (Hesperide) [inline content/en/serverless/installation/nodejs.md:287]:
```suggestion
Datadog’s integration with AWS X-Ray allows you to visualize end-to-end serverless transactions, so you can zero in on the source of any errors or slowdowns, and see how the performance of your functions impacts your users’ experience. Depending on your language and configuration, choose between setting up Datadog APM or the AWS X-Ray integration for your tracing needs. Learn more about configuring distributed tracing [here][4].
```

File: content/en/serverless/installation/python.md (hunk 5 above)
Comment (Hesperide) [inline content/en/serverless/installation/python.md:319]:
```suggestion
Datadog’s integration with AWS X-Ray allows you to visualize end-to-end serverless transactions, so you can zero in on the source of any errors or slowdowns, and see how the performance of your functions impacts your users’ experience. Depending on your language and configuration, [choose between setting up Datadog APM or the AWS X-Ray integration][5] for your tracing needs.
```

File: content/en/serverless/installation/python.md [hunk 10]
```diff
@@ -265,27 +265,29 @@ See the [latest release][4].
...
+[8]: https://docs.datadoghq.com/serverless/forwarder/#experimental-optional
+[9]: https://docs.datadoghq.com/logs/guide/send-aws-services-logs-with-the-datadog-lambda-function/#collecting-logs-from-cloudwatch-log-group
 {{% /tab %}}
 {{< /tabs >}}
 
 ## Explore Datadog Serverless Monitoring
 
 After you have configured your function following the steps above, you can view metrics, logs and traces on the [Serverless Homepage][3].
 
+### Monitor Custom Serverless Metrics
```
Comment (Hesperide) [inline content/en/serverless/installation/python.md:289]:
```suggestion
### Monitor Custom Business Metrics
```

File: content/en/serverless/installation/ruby.md [hunk 11]
```diff
@@ -61,6 +61,8 @@ You need to subscribe the Datadog Forwarder Lambda function to each of your func
 
 After you have configured your function following the steps above, you should be able to view metrics, logs and traces on the [Serverless Homepage][8].
 
+### Monitor Custom Serverless Metrics
```
Comment (Hesperide) [inline content/en/serverless/installation/ruby.md:64]:
```suggestion
### Monitor Custom Business Metrics
```

File: content/en/serverless/installation/python.md [hunk 12]
```diff
@@ -310,6 +312,16 @@ def get_message():
     return "Hello from serverless!"
 ```
 
+Learn more about submitting custom metrics [here][4].
```
Comment (Hesperide) [inline content/en/serverless/installation/python.md:315]:
```suggestion
[Enable custom metric submission][3] to get started.
```

File: content/en/serverless/installation/ruby.md [hunk 13]
```diff
@@ -92,7 +94,13 @@ def some_operation()
     end
 end
 ```
+Learn more about submitting custom metrics [here][9].
```
Comment (Hesperide) [inline content/en/serverless/installation/ruby.md:97]:
```suggestion
[Enable custom metric submission][3] to get started.
```

File: content/en/serverless/installation/ruby.md (hunk 6 above)
Comment (Hesperide) [inline content/en/serverless/installation/ruby.md:101]:
```suggestion
Datadog’s integration with AWS X-Ray allows you to visualize end-to-end serverless transactions, so you can zero in on the source of any errors or slowdowns, and see how the performance of your functions impacts your users’ experience. Depending on your language and configuration, [choose between setting up Datadog APM or the AWS X-Ray integration][5] for your tracing needs.
```

File: content/en/tracing/serverless_functions/_index.md (hunk 8 above)
Comment (Hesperide) [reply content/en/tracing/serverless_functions/_index.md:36]: earlier in the same page

**Review by cswatt — APPROVED** [review]
approved for styleguide

**Review by pinkerton — APPROVED** [review]

**Pull request merged**